File: lesson3/python3/homework/homework3_10d.py
#!/usr/bin/python3
#: lesson3/python3/homework/homework3_10d.py

x = float(input("Введите перевое координата => "))
y = float(input("Введите второе координата => "))

if ((x + 1) * (x + 1) + y*y / 2 >= 1 and
   (x - 1) * (x - 1) + y*y / 2 <= 1 and x >= 0):
    print("попали.")
else:
    print("непопали.")

File: lesson3/c_c++/homework/homework3_10d.c
//: lesson3/c_c++/homework/homework3_10d.c
#include <stdio.h>

int main() {
    float x, y;
    printf("Введите первую координату => ");
    scanf("%f", &x);
    printf("Введине вторую коокдинату => ");
    scanf("%f", &y);

    if ((x+1) * (x+1) + y*y / 2 > 1 &&
            (x-1) * (x-1) + y*y / 2 < 1 && x >= 0) {
        printf("попали.\n");
    } else {
        printf("непопали.\n");
    }
    return 0;
}
